Jenkinsfile_CNP

```diff
@@ -191,7 +191,7 @@ enableSecurityScan()
 
     afterSuccess('akschartsinstall') {
         onPR {
-            if (env.ENABLE_WA == true) {
+            if (env.ENABLE_WA) {
                 githubUpdateDeploymentStatus(githubCreateDeployment("-AIP"), "https://ia-case-api-pr-${CHANGE_ID}-aip-frontend.preview.platform.hmcts.net")
```

Comment:
Will WA be activated by default?

Reply:
There is an issue in the code where sometimes env.ENABLE_WA == true works but sometimes env.ENABLE_WA works, im going to push it into master and see what happens, might need to revert if it fails more appeals in preivew